global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "3.2.0"
   },
   "sdk": {
-    "version": "8.0.100",
+    "version": "9.0.100",
```

Comment:
Note: I don't plan to merge this change via this PR.
I needed this to test NET9 locally.